src/js/helpers/LayersHelper.js

```diff
@@ -29,55 +35,160 @@ const LayersHelper = {
     esriLayer.setVisibleLayers(esriLayer.visibleLayers);
   },
 
-  updateFiresLayerDefinitions (startDate = null, endDate = null, layer, selectValue = null) {
+  updateFiresLayerDefinitions (startDate = null, endDate = null, layer, language, selectValue = null) {
     if (brApp.map) {
       const firesLayer = layer.hasOwnProperty('visibleLayers') ? layer : brApp.map.getLayer(layer.id);
-      const fireID = firesLayer.id === 'VIIRS_ACTIVE_FIRES' ? 'viirs' : 'modis';
+      let layerObj = {};
+      const options = {};
+     
+      const layersPanel = resources.layerPanel.GROUP_LCD.layers;
+      layersPanel.forEach(layerPanel => {
+        if (firesLayer.id.includes(layerPanel.id)) {
+          const index = layersPanel.indexOf(layerPanel);
+          layerObj = resources.layerPanel.GROUP_LCD.layers[index];
+          layerObj.layersId = [21];
+        }
+      });
+     
+      // if (selectValue && layerObj.id.includes('VIIRS_ACTIVE_FIRES')) {
+      //   switch (selectValue) {
+      //     case '0':
+      //       layerObj.id = 'VIIRS_ACTIVE_FIRES_24HR';
+      //       layerObj.layersId = [21];
+      //       break;
+      //     case '1':
+      //       layerObj.id = 'VIIRS_ACTIVE_FIRES_48HR';
+      //       layerObj.layersId = [21];
+      //       break;
+      //     case '2':
+      //       layerObj.id = 'VIIRS_ACTIVE_FIRES_72HR';
+      //       layerObj.layersId = [21];
+      //       break;
+      //     case '3':
+      //       layerObj.id = 'VIIRS_ACTIVE_FIRES_7D';
+      //       layerObj.layersId = [21];
+      //       break;
+      //     case '4':
+      //       layerObj.id = 'VIIRS_ACTIVE_FIRES_1YR';
+      //       layerObj.layersId = [21];
+      //       break;
+      //     default:
+      //       console.log('default');
+      //       break;
+      //   }
+      // }
+      
+      // if (selectValue && layerObj.id.includes('MODIS_ACTIVE_FIRES')) {
+      //   switch (selectValue) {
+      //     case '0':
+      //       layerObj.id = 'MODIS_ACTIVE_FIRES_24HR';
+      //       layerObj.layersId = [21];
+      //       break;
+      //     case '1':
+      //       layerObj.id = 'MODIS_ACTIVE_FIRES_48HR';
+      //       layerObj.layersId = [21];
+      //       break;
+      //     case '2':
+      //       layerObj.id = 'MODIS_ACTIVE_FIRES_72HR';
+      //       layerObj.layersId = [21];
+      //       break;
+      //     case '3':
+      //       layerObj.id = 'MODIS_ACTIVE_FIRES_7D';
+      //       layerObj.layersId = [21];
+      //       break;
+      //     case '4':
+      //       layerObj.id = 'MODIS_ACTIVE_FIRES_1YR';
+      //       layerObj.layersId = [21];
+      //       break;
+      //     default:
+      //       console.log('default');
+      //       break;
+      //   }
+      // }
```

Comment:
@SwampGuzzler Will remove this commented out code once we discuss and make sure the fires bug is fixed

Reply:
Update: I really don't think this is needed anymore, but I'm leaving it in for now.